supervisor/discovery/services/ozw.py

```diff
@@ -1,4 +1,4 @@
-"""Discovery service for Zwave MQTT."""
+"""Discovery service for OpenZwave MQTT."""
```

Comment:
Discovery service for Z-Wave JS Server.

Reply:
Right, I updated also the files from the "old" integration for making the name clear

Reply:
so there will be another service defined for zwave_js, right ?

Reply:
All clear now. Changes to ozw are in the same commit. Ignore my ramblings then!